ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.21-upgrade-from-nightly-4.20.yaml

```diff
@@ -52,6 +52,19 @@ base_images:
     namespace: ocp
     tag: vsphere-csi-driver-operator-test
 releases:
+  arm64-initial:
+    candidate:
+      architecture: arm64
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.21"
```

Comment:
Should intial be 4.20 here and  [here](https://github.com/openshift/release/blob/2ecd7a36d855303701e020001984d136c5efa0e6/ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.21-upgrade-from-nightly-4.20.yaml#L73)?

Reply:
Thanks!  Your question helped raise a question I hadn't asked with this PR, so I'll be making another pass.

Reply:
Updated in new locations that should cover the version issues.  Thanks for catching that!